occurrence-download/src/main/resources/download-workflow/species-list/workflow.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
+<workflow-app name="${occurrence.environment}-species-list-download" xmlns="uri:oozie:workflow:0.4.5">
+<global>
+    <job-tracker>${hadoop.jobtracker}</job-tracker>
+    <name-node>${hdfs.namenode}</name-node>
+    <configuration>
+      <property>
+        <name>oozie.launcher.mapreduce.task.classpath.user.precedence</name>
+        <value>true</value>
+      </property>
+    </configuration>
+  </global>
+
+  <start to="from_hive_download"/>
+
+  <!-- Execute the query using sequential approach -->
+  <action name="from_hive_download">
+    <hive xmlns="uri:oozie:hive-action:0.4">
+      <job-xml>conf/hive-default.xml</job-xml>
+      <script>hive-scripts/execute-species-list-query.q</script>
+      <param>hiveDB=${hive_db}</param>
+      <param>speciesListTable=${download_table_name}</param>
+      <param>whereClause=${hive_query}</param>
+    </hive>
+    <ok to="archive_table"/>
+    <error to="drop_tables_failure"/>
+  </action>
+
+  <action name="archive_table">
+    <java>
+      <main-class>org.gbif.occurrence.download.file.simplecsv.SimpleCsvArchiveBuilder</main-class>
+      <arg>/user/hive/warehouse/${hive_db}.db/${download_table_name}/</arg>
+      <arg>/occurrence-download/${occurrence.environment}-downloads/</arg>
+      <arg>${download_key}</arg>
+      <arg>PRE_DEFLATED</arg>
+      <arg>${download_format}</arg>
+      <capture-output/>
+    </java>
+
+    <ok to="persist_usages"/>
+    <error to="kill"/>
+  </action>
+  
+  <action name="persist_usages">
```

Comment:
since we have 2 java action one after another, we should consider merge them into a wrapper class to avoid the overhead of creating 2 mappers?